js/base/Exchange.js

```diff
@@ -1207,6 +1220,10 @@ module.exports = class Exchange {
             'timeInForce': timeInForce,
             'postOnly': postOnly,
             'trades': trades,
+            'reduceOnly': this.safeValue (order, 'reduceOnly'),
+            'triggerPrice': this.parseNumber (this.safeString2 (order, 'triggerPrice', 'stopPrice')),
```

Comment:
we're missing `stopPrice` here because we still support it

Reply:
I thought the unified name was trigger price though? With this implementation, `stopPrice` will remain there if the subclass contains `stopPrice` in the response, otherwise only `triggerPrice` will be in the response

Reply:
@samgermain it is, but we need to support `stopPrice` for backward compatibility. If we don't add it here, exchanges providing it as a string will return it as a string to the user